app/models/maintenance_tasks/progress.rb

```diff
@@ -50,14 +50,21 @@ def max
     def text
       count = @run.tick_count
       total = @run.tick_total
-      if !total?
-        "Processed #{pluralize(count, "item")}."
-      elsif over_total?
-        "Processed #{pluralize(count, "item")} (expected #{total})."
```

Comment:
Purely stylistic: I think I preferred the old way, with a bit of duplication but a straight if/elsif/else. If we really cared about duplication we could also de-duplicate the call to `"item".pluralize(count)`. I don't really like the local variables.

Reply:
No worries I can change it!